site/docs/components/stepper-input/accessibility.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+## Best practices
+
+- Wrap `StepperInput` in [`FormField`](/salt/components/form-field) to provide the field with a visible label, help text and a status message for validation feedback. This will ensure it complies with accessibility guidelines. You can find more information [in the W3C form instructions](https://www.w3.org/WAI/tutorials/forms/instructions/).
+- `aria-valuenow`, `aria-valuemax`, and `aria-valuemin` attributes are set to provide users with information about the current value, and the allowable range of values.
+
+## Keyboard interactions
+
+Stepper input inherits keyboard interactions from [`Input`](/salt/components/input/accessibility).
+
+The following interactions are specific to Stepper Input.
+
+<KeyboardControls>
+<KeyboardControl keyOrCombos="Up Arrow">
+
+When the input area has focus, increases the number by the increment value (default to 1), up to the maximum value if set.
```

Comment:
Can't remember which is the correct way but this should say "this action increases" or "Up arrow increases"

Reply:
Let's go with "Up arrow increases"